core-blocks/gallery/editor.scss

```diff
@@ -34,6 +34,16 @@
 		width: calc( 100% - 8px );
 		left: 4px;
 		margin-top: -4px;
+
+		// if this inline toolbar has negative margin, it's hidden by the overflow that we need for scrolling long captions
+		.editor-rich-text__inline-toolbar {
+			top: 0;
+		}
+
+		// make extra space for the inline toolbar
+		figcaption {
+			padding-top: 48px;
```

Comment:
Does this have a class name we could target instead? Might be nice. If not no worries, I just always worry about targeting tag names for specificity… and possibly performance but that's super nitpick.

Reply:
It does, and I just pushed a thing to use it. I didn't do that previously because `editor-rich-text__tinymce` is used in a ton of places. But I noticed the Button uses that class to target a similar thing, so I simply used it properly scoped.